### PR TITLE
Fixed issue with alert overlay on map + fixed issue duplicated css due to css imports in js

### PR DESCRIPTION
**Pull request by chrisvanmook**

### CODE REVIEW

File: src/_styles.scss [hunk 1]
```diff
@@ -1,9 +1,4 @@
 @import '~@datapunt/asc-assets/static/fonts/fonts.css';
-/* COMPONENTS */
-@import './map/map';
-
-/* Since we now use code-splitting / chunks, we need to load shared css in here to avoid warnings */
```
Comment (royderks) [inline src/_styles.scss:5]: Is this comment resolved with this pr?

File: src/app/components/NotificationAlert/NotificationAlert.tsx [hunk 2]
```diff
@@ -16,15 +16,6 @@ const Content = styled.div`
   }
 `
 
-const StyledAlertMessage = styled(AlertMessage)`
-  z-index: 3;
-
-  /* Ensure outline is visible when element is in focus */
```
Comment (royderks) [inline src/app/components/NotificationAlert/NotificationAlert.tsx:22]: Is this fix from last week still there?

File: src/_styles.scss (hunk 1 above)
Comment (chrisvanmook) [reply src/_styles.scss:5]: Yes, this comment is obsolete now, as we load every scss in one file now. Let's consider the .scss part legacy css, we're now moving on to styled-components

File: src/app/components/NotificationAlert/NotificationAlert.tsx (hunk 2 above)
Comment (chrisvanmook) [reply src/app/components/NotificationAlert/NotificationAlert.tsx:22]: Reverted, but left out the z-index

**Pull request merged**